README.md

```diff
@@ -138,6 +152,18 @@ The ramifications of changing the `useHandlebars` setting are explained in the s
 
 [Adding banners](#adding-banners) describes the effects of setting `generateBanners` to `true`.
 
+### Define global constants at compile time using
```

Comment:
It seems that this headline is either missing a word or has one word in spare.

Reply:
Oops :D its one word too spare.